tgui/packages/tgui/interfaces/PreferencesMenu/CharacterPreferences/MainPage.tsx

```diff
@@ -578,6 +592,11 @@ export function MainPage(props: MainPageProps) {
                 showGender={
                   currentSpeciesData ? !!currentSpeciesData.sexes : true
                 }
+                canDeleteCharacter={
+                  Object.values(data.character_profiles).filter((name) => name)
```

Comment:
`filter((name) => name)`...? This doesn't do anything

Reply:
Can we make this `!!name` to save future braincells